src/js/index.js

```diff
@@ -360,6 +399,14 @@ function populateDropdown(data, preselected) {
     }
 
     for (var i = 0; i<items.length; i+=2){
+        
+        if (!items[i+1]){
+            continue;
+        }
+     
+        if (!mimeLookup(items[i+1]).startsWith("audio")){
```

Comment:
Any reason why we don't want to support video files as well?
I was thinking that we would enable all file types video or audio that are being supported by ffmpeg

Reply:
Good point! I propose we go through with this PR and open another issue for that matter so that I can implement it after the corresponding ffmpeg filetype support generalization issue is complete on the backend as well and we can test it immediately.